scanner/e2etests/helmchart/templates/db-deployment.yaml

```diff
@@ -0,0 +1,96 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.app.db.name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Values.app.db.name }}
+spec:
+  replicas: 1
+  minReadySeconds: 15
+  selector:
+    matchLabels:
+      app: {{ .Values.app.db.name }}
+  template:
+    metadata:
+      namespace: {{ .Release.Namespace }}
+      labels:
+        app: {{ .Values.app.db.name }}
+    spec:
+      securityContext:
+        fsGroup: 70  # Set the group ownership to the `postgres` group, see dockerfile.
+      initContainers:
+      - name: init-db
+        image: "{{ (printf "%s/%s:%s" .Values.image.repository .Values.app.db.name .Values.image.tag ) }}"
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: POSTGRES_PASSWORD_FILE
+          value: "/run/secrets/stackrox.io/secrets/password"
+        - name: ROX_SCANNER_DB_INIT
```

Comment:
Is this needed?

Reply:
I don't know. Copied from `stackrox/scanner` for safety.